### PR TITLE
Switch from iproxy to tcprelay and use tmp directory for storing Odyssey bootstrap files

### PR DESCRIPTION
**Pull request by marijuanARM**
In theory should work as-is, but isn't tested (which I'll do tomorrow)

### CODE REVIEW

**Comment by NICHOLAS85** [comment]
Just curious, does tcprelay only function with Python 2? I ask because Python 2 has reached EOL and is beginning to be dropped from repos in favor of Python 3. Some users may not be able to install it through their package manager on their Linux installation.

**Comment by M1staAwesome** [comment]
A quick google search does seem to show [this](https://github.com/matteyeux/tcprelay) fork which has python3 support. Python 2 still is installed by default on most major Linux distrbutions though, so I'd probably just stay with that.

File: procursus-deploy-linux-macos.sh [hunk 1]
```diff
@@ -26,16 +35,19 @@ if [[ "${ARM}" = yes ]]; then
 		exit 1
 	fi
 else
-	if which iproxy >> /dev/null; then
-		iproxy 4444 44 >> /dev/null 2>/dev/null &
-	else
-		echo "Error: iproxy not found"
-		exit 1
+	curl -L -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/usbmux.pyc https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/usbmux.py https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/tcprelay.py
```
Comment (NICHOLAS85) [inline procursus-deploy-linux-macos.sh]: Make sure these change to the upstream url before this gets merged

File: procursus-deploy-linux-macos.sh [hunk 2]
```diff
@@ -26,16 +35,20 @@ if [[ "${ARM}" = yes ]]; then
 		exit 1
 	fi
 else
-	if which iproxy >> /dev/null; then
-		iproxy 4444 44 >> /dev/null 2>/dev/null &
-	else
-		echo "Error: iproxy not found"
-		exit 1
+	cd $TMP
+	curl -L -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/usbmux.pyc -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/usbmux.py -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/tcprelay.py
```
Comment (NICHOLAS85) [inline procursus-deploy-linux-macos.sh:39]: Make sure these change to the upstream url before this gets merged

Comment (M1staAwesome) [reply procursus-deploy-linux-macos.sh:39]: completely forgot about that, thanks

**Pull request closed without merging**